sycl/include/sycl/ext/oneapi/experimental/builtins.hpp

```diff
@@ -114,22 +114,22 @@ inline __SYCL_ALWAYS_INLINE
     tanh(sycl::marray<T, N> x) __NOEXC {
   sycl::marray<T, N> res;
 #if defined(__SYCL_DEVICE_ONLY__) && defined(__NVPTX__)
-#define FUNC_VEC native::tanh
-#define FUNC FUNC_VEC
+#define Func_VEC native::tanh
+#define Func Func_VEC
```

Comment:
There are still using identifiers from user's name space. I see they these are used only once, so probably, we should just remove them. Thoughts?

Reply:
Hi @bader I could expand them like this: https://github.com/intel/llvm/pull/8031/commits/d5b5120d7d1353f6938df235abe6efd7237747f3

Reply:
Great. Thanks!